wgpu-core/src/instance.rs

```diff
@@ -649,8 +649,10 @@ crate::impl_storage_item!(Adapter);
 #[derive(Clone, Debug, Error)]
 #[non_exhaustive]
 pub enum GetSurfaceSupportError {
-    #[error("Surface is not supported by the adapter")]
-    Unsupported,
```

Comment:
this looks like a breaking change, so should probably remove this part of https://github.com/gfx-rs/wgpu/pull/6510 again!